buildtest/cli/report.py

```diff
@@ -797,16 +809,28 @@ def report_cmd(args, report_file=None):
     results.print_report(terse=args.terse, noheader=args.no_header, count=args.count)
 
 
-def report_summary(report, pager=None):
+def report_summary(report, pager=None, color=None):
     """This method will print summary for report file which can be retrieved via ``buildtest report summary`` command"""
 
     test_breakdown = report.breakdown_by_test_names()
+    if color is None:
+        table = Table(title="Breakdown by test", header_style="blue")
+        table.add_column("Name", style="cyan")
+        table.add_column("Total Pass", style="green")
+        table.add_column("Total Fail", style="red")
+        table.add_column("Total Runs", style="blue")
+    elif color is not None:
```

Comment:
why use `elif` just use `else since the condition is a negation. First one is None and the second one is not None